dexlib2/src/test/java/org/jf/util/HexTest.java

```diff
@@ -0,0 +1,130 @@
+package org.jf.util;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class HexTest {
+
+    private static final int VALUE = 1234567890;
```

Comment:
Personally i'd just inline this each time, instead of declaring it here.

Reply:
I wanted to show that the value stays constant throughout the tests but I will move it to inline